### DISCUSSION TITLE
“Cytus II & Muse Dash合作活动”即将开始！

### DISCUSSION THREAD

**Discussion by byHydrogen** [share121/inter-knot#87] [其他]
本次联动包括6首歌曲，敬请期待！
![1723194204522](https://github.com/user-attachments/assets/a7762134-4865-4f1a-8a0d-da1775de241b)
律动轨迹的第六个歌曲系列：
“Cytus II & Muse Dash合作活动”即将开始！
#律动轨迹# #Rizline# #Pigeon Games# #鸽游#

**Comment by 132641515Zzy** [comment]
好好好，搁这打广告

**Comment by Tragedysaver1024** [comment]
[涉及不文明用语，已屏蔽]，给我干哪来了这是，这还是绳网吗？
